subprojects/bom/bom.gradle

```diff
@@ -15,7 +15,7 @@ dependencies {
         api(project.rootProject)
         project.rootProject.subprojects.forEach { subproject ->
             if (!subproject.name.endsWith("Test") &&
-                !subproject.name.endsWith("-test") &&
+                !subproject.name.endsWith("java21-test") &&
```

Comment:
The module name needs to change, not the logic.

Rename java21 to java21-test in settings.gradle and folder.

Reply:
Ok. I'll make the required changes.